### PR TITLE
Add clipboard support for Windows

### PR DESCRIPTION
**Pull request by EricApostal**
Release Notes:

- Added Read / Write clipboard support to Windows via copypasta

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,25 +0,0 @@
-.idea
```
Comment (mikayla-maki) [inline .gitignore:1]: Please restore this `.gitignore`

File: crates/gpui/Cargo.toml [hunk 2]
```diff
@@ -72,6 +72,7 @@ usvg = { version = "0.14", features = [] }
 util.workspace = true
 uuid = { version = "1.1.2", features = ["v4"] }
 waker-fn = "1.1.0"
+copypasta = "0.10.1"
```
Comment (mikayla-maki) [inline crates/gpui/Cargo.toml:75]: This should be moved to the `[target.'cfg(any(target_os = "linux", target_os = "windows"))'.dependencies]` section.

**Review by mikayla-maki — CHANGES_REQUESTED** [review]

**Comment by EricApostal** [comment]
These issues should be resolved, sorry about that.

**Comment by mikayla-maki** [comment]
Please also resolve the merge conflicts!

**Comment by EricApostal** [comment]
> Please also resolve the merge conflicts!

I've synced with upstream now :)

**Comment by EricApostal** [comment]
@cla-bot check

**Review by mikayla-maki — APPROVED** [review]

**Comment by mikayla-maki** [comment]
I think you updated the Cargo.lock a bit too much for Zed right now. Try restoring it from main, then running cargo check, and seeing what happens?

**Review by mikayla-maki — APPROVED** [review]

**Comment by mikayla-maki** [comment]
Hmmm, it looks like things are still a bit broken, can you double check? I also pushed a `cargo fmt` to this fork FYI

**Comment by EricApostal** [comment]
I am sorry about that, I messed up my compile target and misunderstood the style check fail. I've fixed the imports and verified that all implemented functionality works as expected on Windows.

**Comment by EricApostal** [comment]
I'd also like to note that I really do apologize for some of the carelessness in this PR- all of these problems should have been avoided ahead of time by doing more research beforehand. I appreciate how patient you've been with me 😄 .

**Comment by mikayla-maki** [comment]
Don't worry at all! Now you know, and can make even better PRs next time :D :D :D

**Pull request merged**